parent/pom.xml

```diff
@@ -45,6 +45,7 @@
         <slf4j.version>1.7.6</slf4j.version>
         <test.oak>false</test.oak>
         <test.vm.options>-Doak=$test.oak}</test.vm.options>
+        <argLine>${test.vm.options}</argLine>
```

Comment:
I'm not sure this is correct. the argline might be used by other plugins as well. I'd rather keep it with the surefire plugin.